DigitalLearningSolutions.Web.AutomatedUiTests/TestHelpers/InputHelper.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace DigitalLearningSolutions.Web.AutomatedUiTests.TestHelpers
+{
+    using OpenQA.Selenium;
+    using OpenQA.Selenium.Support.UI;
+
+    public static class InputHelper

```

Comment:
Nice! Minor thing - could we call this FormHelper as it also includes the button actions/form submission? 👍

Reply:
To be honest, these could also be part of the DriverHelper class, given that they're extensions of it